packages/backend/src/open_payments/payment_pointer/model.ts

```diff
@@ -14,16 +15,26 @@ export class PaymentPointer
     return 'paymentPointers'
   }
 
-  static relationMappings = {
+  static relationMappings = () => ({
     asset: {
       relation: Model.HasOneRelation,
       modelClass: Asset,
       join: {
         from: 'paymentPointers.assetId',
         to: 'assets.id'
       }
+    },
+    keys: {
+      relation: Model.HasManyRelation,
+      modelClass: PaymentPointerKeys,
+      join: {
+        from: 'paymentPointers.id',
+        to: 'paymentPointerKeys.paymentPointerId'
+      }
     }
-  }
+  })
+
+  public keys!: PaymentPointerKeys[]
```

Comment:
I don't think we should do a join on keys with every payment pointer lookup.
```suggestion
  public keys?: PaymentPointerKeys[]
```